content/terraform/v1.12.x/data/language-nav-data.json

```diff
@@ -56,9 +56,26 @@
     ]
   },
   {
-    "title": "Set configuration parameters",
-    "path": "parameterize",
-    "alias": "variables, outputs, locals"
+    "title": "Manage values in modules",
+    "alias": "variables, outputs, locals",
+    "routes": [
+      {
+        "title": "Overview",
+        "path": "values"
```

Comment:
If we do go with the name above, this would be the URL path (which is the same path we previously had for TFv1.1-1.11).